README.md

```diff
@@ -14,7 +14,12 @@ Features:
 * small and easy to use
 
 To use:
-* open "moksha.html" in a modern browser
```

Comment:
It didn't work in IE9 when I tried it today. Did it ever work in IE?

On Tue, Jan 6, 2015 at 10:34 AM, Ryan McDonald <notifications@github.com>
wrote:

> In README.md
> <https://github.com/devops001/moksha_web/pull/2#discussion-diff-22540927>:
>
> > @@ -14,7 +14,12 @@ Features:
> >  * small and easy to use
> >
> >  To use:
> > -* open "moksha.html" in a modern browser
>
> it should still work in other browsers
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/devops001/moksha_web/pull/2/files#r22540927>.
>